index.html

```diff
@@ -72,6 +76,191 @@ <h1>
+              an existing story over the course of a day without spamming their
+              friends.
+            </p>
+            <ul class="frameworks-list work-child">
+              <li class="framework">html</li>
+              <li class="framework">Ruby on rails</li>
+              <li class="framework">css</li>
+              <li class="framework">javaScript</li>
+            </ul>
+            <button type="button" class="project-btn work-child">See Project</button>
+          </div>
+        </div>
+        <div class="wraper">
+          <div class="image">
+            <img alt="work3" src="./assets/imgs/Snapshoot3.png" />
+          </div>
+          <div class="work-info">
+            <h2 class="work-child">Facebook 360</h2>
+            <div class="description-wraper work-child">
+              <h3 class="project-name">FACEBOOK</h3>
+              <img alt="dot" src="./assets/imgs/Counter.png" />
+              <p>Full Stack Dev</p>
+              <img alt="dot" src="./assets/imgs/Counter.png" />
+              <p>2015</p>
+            </div>
+            <p class="work-text work-child">
+              Exploring the future of media in Facebook's first Virtual Reality
+              app; a place to discover and enjoy 360 photos and videos on Gear
+              VR.
+            </p>
+            <ul class="frameworks-list work-child">
+              <li class="framework">html</li>
+              <li class="framework">Ruby on rails</li>
+              <li class="framework">css</li>
+              <li class="framework">javaScript</li>
+            </ul>
+            <button type="button" class="project-btn work-child">See Project</button>
+          </div>
+        </div>
+        <div class="wraper second">
+          <div class="image">
+            <img alt="work4" src="./assets/imgs/Snapshoot4.png" />
+          </div>
+          <div class="work-info">
+            <h2 class="work-child">Uber Navigation</h2>
+            <div class="description-wraper work-child">
+              <h3 class="project-name">Uber</h3>
+              <img alt="dot" src="./assets/imgs/Counter.png" />
+              <p>Lead Developer</p>
+              <img alt="dot" src="./assets/imgs/Counter.png" />
+              <p>2018</p>
+            </div>
+            <p class="work-text work-child">
+              A smart assistant to make driving more safe, efficient, and fun by
+              unlocking your most expensive computer: your car.
+            </p>
+            <ul class="frameworks-list work-child">
+              <li class="framework">html</li>
+              <li class="framework">Ruby on rails</li>
+              <li class="framework">css</li>
+              <li class="framework">javaScript</li>
+            </ul>
+            <button type="button" class="project-btn work-child">See Project</button>
+          </div>
+        </div>
+      </section>
+      <div id="container2">
+        <section id="about-me">
```

Comment:
[Optional] Kindly consider adding a left margin to the about myself section. It will make your page look the same like the Figma design. 
![image](https://user-images.githubusercontent.com/62849872/143661886-751cc0fd-a0f6-40f0-9e03-66a94f7f4866.png)